ord_schema/validations_test.py

```diff
@@ -0,0 +1,86 @@
+"""Tests for ord_schema.units."""
+
+from absl.testing import absltest
+from absl.testing import parameterized
+
+from ord_schema import validations
+from ord_schema.proto import ord_schema_pb2 as schema
+
+
+class UnitsTest(parameterized.TestCase, absltest.TestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.validations = validations
```

Comment:
Just use `validations` directly in the methods; not `self.validations`.

Reply:
I'm not sure why I thought that was the right thing to do